context/app/static/js/components/workspaces/AdvancedConfigOptions/AdvancedConfigOptions.tsx

```diff
@@ -0,0 +1,185 @@
+
+    for (let i = start; i <= convertedMax; i += markInterval) {
+      tempMarks.push({ value: i, label: i });
+    }
+
+    return tempMarks;
+  }, [convertedMin, convertedMax, markInterval]);
+
+  return (
+    <Stack marginTop={1}>
+      <Stack direction="row" spacing={1} alignItems="center">
+        <StyledSubtitle>{label}</StyledSubtitle>
+        <InfoTooltipIcon iconTooltipText={tooltip} />
+      </Stack>
+      <Stack padding={1}>
+        <Slider
+          value={convert(field.value[id] as number)}
+          onChange={(e, value) =>
+            field.onChange({
+              ...field.value,
+              [id]: unconvert(value as number),
+            })
+          }
+          valueLabelDisplay="auto"
+          marks={marks}
+          min={convertedMin}
+          max={convertedMax}
+        />
+      </Stack>
+    </Stack>
+  );
+}
+
+const configSliderOptions: Omit<ConfigSliderProps<Record<string, number>>, 'field'>[] = [
+  {
+    id: 'time_limit_minutes',
+    label: 'Time Limit (hours)',
+    tooltip: 'Session duration for your workspace.',
+    min: MIN_TIME_LIMIT_MINUTES,
+    max: MAX_TIME_LIMIT_MINUTES,
+    conversionFactor: 60,
+  },
+  {
+    id: 'memory_mb',
+    label: 'Memory (GB)',
+    tooltip: 'Available memory for your workspace.',
+    min: MIN_MEMORY_MB,
+    max: MAX_MEMORY_MB,
+    conversionFactor: 1000,
+    markInterval: 2,
+  },
+  {
+    id: 'num_cpus',
+    label: 'Number of CPUs',
+    tooltip: 'Number of CPUs available for your workspace.',
+    min: MIN_NUM_CPUS,
+    max: MAX_NUM_CPUS,
+  },
+];
+
+type WorkspaceJobTypeFieldProps<FormType extends FieldValues> = Pick<UseControllerProps<FormType>, 'control'>;
+
+function AdvancedConfigOptions<FormType extends FieldValues>({
+  control,
+  description,
+}: WorkspaceJobTypeFieldProps<FormType> & { description: string }) {
+  const { field } = useController({
+    name: 'workspaceResourceOptions' as Path<FormType>,
+    control,
+    rules: { required: true },
+  });
+
+  const isDefault =
+    field.value.time_limit_minutes === DEFAULT_TIME_LIMIT_MINUTES &&
+    field.value.memory_mb === DEFAULT_MEMORY_MB &&
+    field.value.num_cpus === DEFAULT_NUM_CPUS &&
+    field.value.gpu_enabled === DEFAULT_GPU_ENABLED;
+
+  const handleRestoreDefaults = useCallback(() => {
+    field.onChange({
+      time_limit_minutes: DEFAULT_TIME_LIMIT_MINUTES,
+      memory_mb: DEFAULT_MEMORY_MB,
+      num_cpus: DEFAULT_NUM_CPUS,
+      gpu_enabled: DEFAULT_GPU_ENABLED,
+    });
+  }, [field]);
+
+  return (
+    <StyledAccordion>
+      <AccordionSummary expandIcon={<ArrowDropDownRounded color="primary" />}>
+        <StyledHeader>Advanced Configurations (Optional)</StyledHeader>
+      </AccordionSummary>
+      <AccordionDetails>
+        <Stack>
+          <Typography>{description}</Typography>
+          <SpacedSectionButtonRow
+            buttons={
+              <Stack direction="row" gap={1} marginTop={1}>
```

Comment:
What is the `gap` prop for? Does this need to be a `Stack` could it be a `Box` or just remove the wrapper component altogether?

Reply:
This was an issue with positioning the button on the far right - removed wrapper and fixed with `alignSelf="flex-end"`.